spec/views/my/_list_works.html.erb_spec.rb

```diff
@@ -4,15 +4,17 @@
 
   let(:work_title) { 'Work Title' }
   let(:coll_title) { 'Collection Title' }
-  let(:collection) { FactoryGirl.create(:collection, title: coll_title) }
-  let!(:work) { FactoryGirl.create(:work, title: [work_title], collections: [collection]) }
+  let(:collection) { FactoryGirl.build(:collection, id: '3197z497t', title: coll_title) }
+  let!(:work) { FactoryGirl.build(:work, id: '3197z511f', title: [work_title]) }
```

Comment:
@jcoyne How does the work get assigned to the collection here?  what am I missing?